content/integrations/AWS-Integration/AWS-CloudFormation-Installation.md

```diff
@@ -12,31 +12,22 @@ This installation method leverages a CloudFormation script which can be viewed [
 
-## Create an AWS Integration in CloudWisdom
-1. Open CloudWisdom and navigate to Integrations > AWS.  
-2. Create a new AWS integration. Ensure that the below tabled properties are set according to your needs.
+## 1. Create an AWS Integration in CloudWisdom
+1. Open CloudWisdom and navigate to **Integrations** > **Amazon Web Services**.  
+2. Select **+ Add Integration**.
+3. Enable the **Cost Explorer API**.
+4. Select the following Detailed Billing Source: **Cost And Usage Report (Recommended)**.
+![cost-explorer-CUR](/images/AWS-CloudFormation-Installation/cost-explorer-cur.png)
+5. Select the **CloudFormation script** link under Configure AWS Permissions. This opens a new tab in AWS.  
+6. Check **I acknowledge that AWS CloudFormation might create IAM resources**.  
+7. Select **Create Stack**. This process may take a few minutes. Wait for the stack to say **CREATE_COMPLETE** before proceeding to the next section.
 
-| AWS Toggle   | Description |
-|--------------|---------------------------------------------------------------|
-| CostExplorer | Captures billing data. This should only be enabled on 1 AWS account, the master billing data account. |
-| CloudWatch   | Enable the CloudWatch toggle to monitor performance data. This can be done on multiple accounts. |  
-![Enable CloudWatch](/images/AWS-CloudFormation-Installation/enable-cloudwatch.png)
-3. For AWS Authentication, select **IAM Role**.  
-![IAM Role](/images/AWS-CloudFormation-Installation/iam-role.png)
-4. Click the **CloudFormation script** link under Configure AWS Permissions. This opens a new tab in AWS.  
-5. Check **I acknowledge that AWS CloudFormation might create IAM resources**.  
-6. Click **Create Role**.  
-![I acknowledge](/images/AWS-CloudFormation-Installation/i-acknowledge.png)
-7. It may take a few minutes to create the role. When ready, the status turns green.
-![Read Only Status Complete](/images/AWS-CloudFormation-Installation/read-only-status-complete.png)
-
-## Add Role ARN to the New AWS Integration
-1. In the AWS console, navigate to **Services** > **IAM** > **Roles**.  
-2. Find the role created in the previous section using the search bar.  
-![Create Role](/images/AWS-CloudFormation-Installation/create-role.png)
-3. Expand **Outputs** and copy the **Role ARN**.  
- - Do **not** copy the _Stack ID_. The Role ARN looks like: `arn:aws:iam::<account-number>:role/Metricly-Role-Name>`
-4. Return to the CloudWisdom.   
+## 2.  Add RoleARN to the New AWS Integration
+1. Select the stack you have created. It will start with `Metricly-Read-Only-Role-`.
+2. Navigate to the **Outputs** tab.
+3. Copy the **Role ARN Value**.
+![role-arn-value](/images/onboarding-wizard/role-arn-value.png)
+4. Return to CloudWisdom.   
 5. Input the Role ARN into **IAM Role ARN**.  
-![IAM Role ARN](/images/AWS-CloudFormation-Installation/iam-role-arn.png)
-6. Click **Save**.  
+![iam-role-arn](/images/AWS-CloudFormation-Installation/iam-role-arn.png)
+6. **Save**.  
```

Comment:
Why Don't we have Detailed Billing Portion here?
<img width="748" alt="Screen Shot 2020-01-17 at 16 23 28" src="https://user-images.githubusercontent.com/35566439/72619377-c773c080-3945-11ea-80d3-608fc9e370c0.png">

Reply:
I can add a link to the CUR page to this document. I worry that if we repeat steps across all of these pages it will be more to maintain/update when AWS changes their UI again haha.  I could also specify the scope of this is just running the cloudformation script? and doesn't include the rest of the setup steps.

Reply:
added a link under step 1.4 for CUR

Reply:
yea, right: if I create a second integration, Detailed Billing Settings are not needed. Maybe needs to be discussed with Brian and Ryan in Slack